recipes/urduhack/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "urduhack" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 66147eb1eb602cf0d63bf832472983f1bd514b58b5ae61b0cee3aa1f12f02631
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - pip
+    - python
+  host:
+    - pip
+    - python
+    - numpy
+  run:
+    - python
+    - tensorflow
+    - transformers
+    - tensorflow-datasets
+    - Click
+    - regex
+
+test:
+  imports:
+    - urduhack
+
+about:
+  home: https://github.com/urduhack/urduhack
+  license: MIT
+  license_file: LICENSE.txt
+  summary: "NLP library for Urdu Language"
+  doc_url: https://urduhack.readthedocs.io/en/stable/
+  dev_url: https://github.com/urduhack/urduhack
+
+extra:
+  recipe-maintainers:
```

Comment:
@muhammadfahid51 What about yourself? Are you going to be a maintainer of this feedstock?

Reply:
@xhochy I will be the maintainer for this feedstock. 

Username: akkefa

Reply:
@xhochy akkefa will be the maintainer of this recipe